.github/workflows/manypkg.yml

```diff
@@ -0,0 +1,46 @@
+name: Manypkg Check
```

Comment:
This name doesn't tell too much to someone who is not familiar with `manypkg`. Maybe something like this? 👇 Perhaps should also change the filename accordingly.

```suggestion
name: Monorepo integrity
```

Reply:
Done, file has been renamed!